sdk/core/core-http/lib/policies/proxyPolicy.ts

```diff
@@ -13,6 +13,32 @@ import { WebResource } from "../webResource";
 import { Constants } from "../util/constants";
 import { URLBuilder } from "../url";
 
+/**
+ * Options to configure a proxy for outgoing requests.
+ */
+
+export interface ProxyOptions {
```

Comment:
We have `ProxySettings` which is of the same shape. Is this replacing that? Or meant to be used differently?